requirements-dev.in

```diff
@@ -5,3 +5,4 @@ isort
 pytest
 pytest-cov
 pytest-django
+factory-boy
```

Comment:
also, didn't you get on board with the alphabetical order? 😅 😝

Reply:
🤦‍♂ fixed.